students/rgpag/lesson03/slicingLab.py

```diff
@@ -0,0 +1,52 @@
+def exchange_first_last(seq):
+	""" return seq with first and last elements exchanged"""
+	L=len(seq)
+	return seq[-1]+seq[1:L-1]+seq[0]
```

Comment:
The call the `len` is necessary since you can use negative index in your slice. This code will work to concatentate a string but for other sequences it will throw a `TypeError` if you try to add, for example, an integer and a list